packages/xo-web/src/xo-app/host/tab-patches.js

```diff
@@ -266,9 +266,12 @@ export default class TabPatches extends Component {
       buttons: [{ label: _('installAllPatchesRedirect'), value: 'goToPool' }],
       icon: 'host-patch-update',
       title: _('installAllPatchesTitle'),
-    }).then(() =>
-      this.context.router.push(`/pools/${this.props.host.$pool}/patches`)
-    )
+    }).then(() => {
+      const { $pool: pool, productBrand } = this.props.host
+      return productBrand === 'XCP-ng'
+        ? installAllPatchesOnPool({ pool })
```

Comment:
It can't say "Go to pool" and then install the patches. Simply call `installAllPatchesOnPool` when it's an XCP-ng.